Cargo.toml

```diff
@@ -4,7 +4,10 @@ version = "0.1.0"
 authors = ["Geoffroy Couprie <geo.couprie@gmail.com>"]
 
 [dependencies]
-nom = "~1.2.0"
+
+[dependencies.nom]
+version = "2.0"
+features = ["verbose-errors"]
```

Comment:
This is just for nicer debugging right now, the feature should go away before merging